contestant_containers/arch/files/supervisord.conf

```diff
@@ -0,0 +1,42 @@
+[supervisord]
+user=root
+nodaemon=true
+logfile=/var/log/supervisord.log
+loglevel=debug
+
+[inet_http_server]
+port=127.0.0.1:9001
+
+[program:sshdir]
```

Comment:
before sshd can start, keys need to be generated.  Some distros generate them during ssh package install, but we super don't want to be shipping ssh keys during build which is why we wipe them in the other Dockerfiles
https://github.com/rfhs/rfctf-container/blob/main/contestant_containers/kali/Dockerfile#L21

For debian based distros we generate the keys at runtime like this https://github.com/rfhs/rfctf-container/blob/main/contestant_containers/kali/files/supervisord.conf#L9-L16 but for non-debian based distros it can easily be done manually https://github.com/rfhs/rfctf-container/blob/main/contestant_containers/pentoo/files/supervisord.conf#L15-L21